.readthedocs.yaml

```diff
@@ -0,0 +1,30 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "mambaforge-4.10"
+  jobs:
+    pre_build: # Generate api stub files before building
+      - make -C doc api
+
+# Build documentation in the doc/ directory with Sphinx
+sphinx:
+   configuration: doc/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
+# formats:
+#    - pdf
+
+# Optionally declare the Python requirements required to build your docs
+conda:
+  environment: environment.yml
```

Comment:
Do you think we should just install the sphinx-related libraries needed for building the docs? We used to install everything on Vercel because it's quite fast, but Readthedocs is a bit slower it seems, so less libraries might speed things up.

Reply:
Done in d21acf9519c2b34fd2ad569014a86f0cb39a4c9a.

Slightly faster. Now it takes 12 minutes.